3.test_cases/10.FSDP/0.create_conda_env.sh

```diff
@@ -10,12 +10,19 @@ chmod +x Miniconda3-latest-Linux-x86_64.sh
 
 source ./miniconda3/bin/activate
 
-conda create -y -p ./pt_fsdp python=3.10
+conda create -y -p ./pt_fsdp python=3.11
 
 source activate ./pt_fsdp/
 
-# Install AWS Pytorch, see https://aws-pytorch-doc.com/
-conda install -y pytorch=2.2.0 pytorch-cuda=12.1 aws-ofi-nccl=1.7.4 torchvision torchaudio transformers datasets fsspec=2023.9.2 --strict-channel-priority --override-channels -c https://aws-ml-conda.s3.us-west-2.amazonaws.com -c nvidia -c conda-forge
+# Set true to install AWS Pytorch. see https://aws-pytorch-doc.com/
+use_aws_pytorch=true
+
+if $use_aws_pytorch; then
+    conda install -y pytorch=2.3.0 pytorch-cuda=12.1 aws-ofi-nccl torchvision torchaudio transformers datasets fsspec=2023.9.2 --strict-channel-priority --override-channels -c https://aws-ml-conda.s3.us-west-2.amazonaws.com -c nvidia -c conda-forge
+else
+    conda install -y pytorch=2.4.1 torchvision torchaudio transformers datasets fsspec=2023.9.2 pytorch-cuda=12.1 -c pytorch -c nvidia
+    conda install -y aws-ofi-nccl=1.9.1 -c https://aws-ml-conda.s3.us-west-2.amazonaws.com -c conda-forge
```

Comment:
Why there is no aws ofi nccl 1.11.0 version yet?

```suggestion
    conda install -y aws-ofi-nccl=1.11.0 -c https://aws-ml-conda.s3.us-west-2.amazonaws.com -c conda-forge
```

Reply:
@mhuguesaws , aws-ofi-nccl=1.11.0 is not available from the conda package channel. Latest available version is 1.9.1. Other newer version requires building package from source code.
I am going to initiate a thread with the team to discuss upgrading the version on the conda channel, but 1.9.1 is the latest for now.